.mailmap

```diff
@@ -0,0 +1,122 @@
+#
+#   - Fix just the name associated with an email address:
+#
+#         New Full Name <email@address>
+#
+#     Example: to map from "AliceProxy <alicewasko@datawire.io>" to
+#     "Alice Wasko <alicewasko@datawire.io>", I'd write
+#
+#         Alice Wasko <alicewasko@datawire.io>
+#
+#   - Adjust an email address (and possibly name as well):
+#
+#         New Full Name <new.email@address> <old.email@address>
+#
+#     Example: to map from "AliceProxy <aliceproxy@protonmail.com>" to
+#     "Alice Wasko <alicewasko@datawire.io>", I'd write
+#
+#         Alice Wasko <alicewasko@datawire.io> <aliceproxy@protonmail.com>
+#
+# In `git log` `--format` strings: the uppercase '%cN', '%cE', '%aN',
+# and '%aE' obey .mailmap; while the lowercase '%cn', '%ce', '%an',
+# and '%ae' ignore it.
+#
+# See the gitmailmap(5) man-page for more information.
+
+# Current Ambassador Labs employees
+Alex Gervais <alex@datawire.io>
+Alex Gervais <alex@datawire.io>                        <alex.gervais@gmail.com>
+Alex Gervais <alex@datawire.io>                        <alexandre.gervais@appdirect.com>
+Alice Wasko <alicewasko@datawire.io>
+Alice Wasko <alicewasko@datawire.io>                   <22459179+AliceProxy@users.noreply.github.com>
+Alice Wasko <alicewasko@datawire.io>                   <aliceProxy@protonmail.com>
+Alice Wasko <alicewasko@datawire.io>                   <aliceproxy@pm.me>
+Alice Wasko <alicewasko@datawire.io>                   <aliceproxy@protonmail.com>
+Andres Osorio <aosorio@datawire.io>                    <95947712+aosoriodw@users.noreply.github.com>
+Bjorn Freeman-Benson <bjorn@datawire.io>               <157783+bnfb@users.noreply.github.com>
+Bjorn Freeman-Benson <bjorn@datawire.io>               <bjorn@Bjorns-MacBook-Pro.local>
+Cindy Mullins <cindymullins@datawire.io>               <57576741+cindymullins-dw@users.noreply.github.com>
+Kelsey Evans <kelsey@datawire.io>
+Kevin Lambert <klambert@datawire.io>                   <kevin.lambert.ca@gmail.com>
+Lance Austin <laustin@datawire.io>                     <lanceaustin26@gmail.com>
+Nick Powell <nickpowell@datawire.io>                   <35627686+njayp@users.noreply.github.com>
+Nick Powell <nickpowell@datawire.io>                   <nickjaypowell@gmail.com>
+Rafael Schloming <rhs@datawire.io>                     <rhs@alum.mit.edu>
+Thomas Hallgren <thomas@datawire.io>                   <thomas@tada.se>
+William Hardin <williamhardin@datawire.io>             <47009048+w-h37@users.noreply.github.com>
+
+# Former Ambassador Labs employees (without a new address)
+Alix Cook <alixcook@datawire.io>                       <73843761+acookin@users.noreply.github.com>
+Douglas Camata <dcamata@datawire.io>                   <159076+douglascamata@users.noreply.github.com>
+Leonardo Luz Almeida <leonardoluz@datawire.io>         <leoluz@users.noreply.github.com>
+Matt McClure <mattmcclure@datawire.io>                 <73489204+mattmcclure-dw@users.noreply.github.com>
```

Comment:
So... we don't have an updated way to reach these people? Is it worth trying to find one?

Reply:
I put a little bit of effort in, these are the few folks that I didn't find one for. I guess the next step would be reaching out on LinkedIn?

Reply:
And to be clear: I didn't look for former-employees in general; this is just the ones that used multiple emails to commit with.

Reply:
OK, looking through the other `@datawire.io` emails: other former-employees without new addresses are Bruce Horn, David Dymko, and Ryan Park.

Reply:
OK, I've reached out to 5 of those 6 people on LinkedIn... I couldn't find Ryan. Or rather, I found a whole bunch of Ryan Parks but couldn't quickly tell which one is him.